vpr/src/route/rr_graph_storage.h

```diff
@@ -216,6 +216,10 @@ class t_rr_graph_storage {
     short node_track_num(RRNodeId id) const; //Same as ptc_num() but checks that type() is consistent
     short node_class_num(RRNodeId id) const; //Same as ptc_num() but checks that type() is consistent
 
+    inline void prefetch_node(RRNodeId id) const {
```

Comment:
Should comment what these prefetchs are.

Reply:
Done